ruby/ext/google/protobuf_c/message.c

```diff
@@ -419,6 +419,13 @@ VALUE Message_to_h(VALUE _self) {
       msg_value = Map_to_h(msg_value);
     } else if (upb_fielddef_label(field) == UPB_LABEL_REPEATED) {
       msg_value = RepeatedField_to_ary(msg_value);
+
+      if (upb_fielddef_msgsubdef(field) != NULL) {
```

Comment:
I think it would be a bit clearer to write:

```
if (upb_fieldddef_type(field) == UPB_TYPE_MESSAGE)
```

Reply:
Makes sense! I fixed that.